src/raftstore/store/worker/compact.rs

```diff
@@ -238,7 +239,7 @@ fn collect_ranges_need_compact(
         }
     }
 
-    // Save the last range that need compacting.
+    // Saves the last range that needs to be compacted.
```

Comment:
Looks like we don't need to change Save to Saves for normal comments. It is only applicable to `///` comments.

e.g.:

`s` for `///`: https://github.com/rust-lang/regex/blob/master/src/dfa.rs#L767
no `s` for normal comments: https://github.com/rust-lang/regex/blob/master/src/dfa.rs#L749

Reply:
/cc @CaitinChen

Reply:
So, how to write normal comments? 😂 @CaitinChen

Reply:
@Hoverbear @brson I also feel confused about this.

Reply:
I agreed with @breeswish that Rust didn't mandate which tense to use in normal comments, and this change isn't necessary. In rust-lang/rust, normal comments are usually written in first-person voice ("we"), explaining the rationale of following code from the writers' perspectives.

If we want to fix on the grammar of normal comments, this should be included as part of CONTRIBUTING.md (normal comment isn't discussed much in https://web.archive.org/web/20170607051152/http://aturon.github.io:80/style/comments.html).

Reply:
I haven't thought about this question before, but I think normal comments can be written in whatever way makes most sense. It does strike me that the imperative voice works fine for inline comments about imperative code, and that's what I tend to use. The old guidelines document @kennytm linked to seems right for doc comments.

Reply:
@breeswish @brson @kennytm Thank you all.